prisma/schema.prisma

```diff
@@ -10,6 +10,16 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model User {
+  id        String    @id @default(uuid())
+  email     String?   @unique
```

Comment:
can email be optional actually? I think there is always one, no matter if its custom login or auth?

Reply:
it can because when you use without an account you don't have an email

Reply:
in dev mode and demo mode? maybe we should just have a placehoder and avoid checking everywhere if there is an email? Just an idea